ControlPanel.tscn

```diff
@@ -0,0 +1,22 @@
+[gd_scene load_steps=2 format=2]
```

Comment:
Chiiiiidoo, we can add more ui elements (buttons and stuff, we could add pause and options for example) inside this node. Controlling them with an script from this node makes sense to me